### ISSUE TITLE
Output Channel Size

### ISSUE THREAD

**Issue by vpapaioannou** [jvanvugt/pytorch-unet#7]
In Unet definition, (last): Conv2d(64, 2, kernel_size=(1, 1), stride=(1, 1)) but, shouldn't be
(last): Conv2d(64, 1, kernel_size=(1, 1), stride=(1, 1))? Aren't we expecting one image in the end? Could you explain?

Thanks.

**Comment by jvanvugt** [comment]
If you are doing binary classification, you can either use a single output channel with sigmoid activation or 2 outputs with softmax activation.
